### PR TITLE
Handles AUS zones in URLs

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

Handles people going to a bookmarked URL for Australia zones - turns out we never got around to implement that 😅 

### Double check

- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

(I for sure though we already did this...)

File: web/src/features/panels/LeftPanel.tsx [hunk 1]
```diff
@@ -39,11 +39,23 @@ function HandleLegacyRoutes() {
...
 function ValidZoneIdGuardWrapper({ children }: { children: JSX.Element }) {
+  const [searchParameters] = useSearchParams();
   const { zoneId } = useParams();
 
   if (!zoneId) {
     return <Navigate to="/" replace />;
   }
+
+  // Handle legacy Australia zone names
+  if (zoneId.startsWith('AUS-')) {
```
Comment (madsnedergaard) [inline web/src/features/panels/LeftPanel.tsx:50]:
```suggestion
  if (zoneId.startsWith('AUS')) {
```

File: web/src/features/panels/LeftPanel.tsx [hunk 2]
```diff
@@ -39,11 +39,23 @@ function HandleLegacyRoutes() {
...
+    return (
+      <Navigate
+        to={`/zone/${zoneId.replace('AUS-', 'AU-')}?${searchParameters}`}
```
Comment (madsnedergaard) [inline web/src/features/panels/LeftPanel.tsx:53]:
```suggestion
        to={`/zone/${zoneId.replace('AUS', 'AU')}?${searchParameters}`}
```

[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/features/panels/LeftPanel.tsx]: Fair point, fixed now :)

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

(but we broke prettier 😅)

**Comment by madsnedergaard** [comment]
> LGTM! 🎉
> 
> (but we broke prettier 😅)

So much for being too lazy to switch branches 😅

**Pull request merged**